language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSPackageCache.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
2018

Reply:
@hevayo This class is being renamed only. Hence did not change the year